### ISSUE TITLE
Can we have first name last name instead of username? 

### ISSUE THREAD

**Issue by zaverichintan** [GravityProject/gravity#108]
This will allow users to know other users by name, username is not very friendly. 
Any suggestions?

**Comment by pmuens** [comment]
Good idea. Maybe we can make this optional so that users who want to protect their privacy are not forced to reveal their real name

**Comment by zaverichintan** [comment]
Yes, making optional is a better work around. 
I believe that social network establishes connection between people, so knowing names would be appreciated
